addons/cert-manager/cert-manager.yaml

```
@@ -27,7 +27,7 @@ spec:
   chartReference:
     chart: cert-manager-setup
     repo: https://mesosphere.github.io/charts/staging
-    version: 0.1.17
+    version: 0.2.1
```

```suggestion
    version: 0.2.0
```
I think the latest version is [`0.2.0`](https://github.com/mesosphere/charts/blob/master/staging/cert-manager-setup/Chart.yaml#L4)

---

(nvm on this since we've since published a newer version `0.2.1`)